src/main/scala/firrtl/annotations/JsonProtocol.scala

```diff
@@ -266,9 +278,14 @@ object JsonProtocol {
     writePretty(safeAnnos)
   }
 
+  /** Deserialize JSON input into a Seq[Annotation]
+    *
+    * @param in JsonInput, can be file or string
+    * @return
+    */
   def deserialize(in: JsonInput): Seq[Annotation] = deserializeTry(in).get
 
-  def deserializeTry(in: JsonInput): Try[Seq[Annotation]] = Try({
+  def deserializeTry(in: JsonInput): Try[Seq[Annotation]] = Try {
```

Comment:
I don't see the paren after the `Try` not sure why, I'll call this done

Reply:
This isn't a comment about the deleted parentheses (I have no strong opinion about the stylistic choice either way), but rather that we need another argument to this function in case the flag was passed on the command line.